pkg/apis/core/types_shoot.go

```diff
@@ -165,6 +165,9 @@ type ShootCredentialsRotation struct {
 	Kubeconfig *ShootKubeconfigRotation
 	// SSHKeypair contains information about the ssh-keypair credential rotation.
 	SSHKeypair *ShootSSHKeypairRotation
+	// ObservabilityUser contains information about the observability-user credential rotation.
+	// +optional
+	ObservabilityUser *ShootObservabilityUserRotation `json:"observabilityUser,omitempty" protobuf:"bytes,4,opt,name=observabilityUser"`
```

Comment:
```suggestion
	Observability *ShootObservabilityRotation `json:"observabilityUser,omitempty" protobuf:"bytes,4,opt,name=observabilityUser"`
```

Let's remove `User` from the name